src/node/internal/internal_buffer.ts

```diff
@@ -0,0 +1,2098 @@
+  buf[offset + 2] = value;
+  value = value >>> 8;
+  buf[offset + 1] = value;
+  value = value >>> 8;
+  buf[offset] = value;
+  return offset + 4;
+}
+
+function writeU_Int24BE(
+    buf: Buffer,
+    value: number,
+    offset: number,
+    min: number,
+    max: number) {
+  value = +value;
+  validateOffset(offset, "offset", 0, buf.length);
+  checkInt(value, min, max, buf, offset, 2);
+
+  buf[offset + 2] = value;
+  value = value >>> 8;
+  buf[offset + 1] = value;
+  value = value >>> 8;
+  buf[offset] = value;
+  return offset + 3;
+}
+
+function validateOffset(
+  value: number,
+  name: string,
+  min: number = 0,
+  max: number = Number.MAX_SAFE_INTEGER,
+) {
+  if (typeof value !== "number") {
+    throw new ERR_INVALID_ARG_TYPE(name, "number", value);
+  }
+  if (!Number.isInteger(value)) {
+    throw new ERR_OUT_OF_RANGE(name, "an integer", value);
+  }
+  if (value < min || value > max) {
+    throw new ERR_OUT_OF_RANGE(name, `>= ${min} && <= ${max}`, value);
+  }
+}
+
+function writeU_Int48LE(
+    buf: Buffer,
+    value: number,
+    offset: number,
+    min: number,
+    max: number) {
+  value = +value;
+  validateOffset(offset, "offset", 0, buf.length);
+  checkInt(value, min, max, buf, offset, 5);
+
+  const newVal = Math.floor(value * 2 ** -32);
+  buf[offset++] = value;
+  value = value >>> 8;
+  buf[offset++] = value;
+  value = value >>> 8;
+  buf[offset++] = value;
+  value = value >>> 8;
+  buf[offset++] = value;
+  buf[offset++] = newVal;
+  buf[offset++] = newVal >>> 8;
+  return offset;
+}
+
+function writeU_Int40LE(
+    buf: Buffer,
+    value: number,
+    offset: number,
+    min: number,
+    max: number) {
+  value = +value;
+  validateOffset(offset, "offset", 0, buf.length);
+  checkInt(value, min, max, buf, offset, 4);
+
+  const newVal = value;
+  buf[offset++] = value;
+  value = value >>> 8;
+  buf[offset++] = value;
+  value = value >>> 8;
+  buf[offset++] = value;
+  value = value >>> 8;
+  buf[offset++] = value;
+  buf[offset++] = Math.floor(newVal * 2 ** -32);
+  return offset;
+}
+
+function writeU_Int32LE(
+    buf: Buffer,
+    value: number,
+    offset: number,
+    min: number,
+    max: number) {
+  value = +value;
+  validateOffset(offset, "offset", 0, buf.length);
+  checkInt(value, min, max, buf, offset, 3);
+
+  buf[offset++] = value;
+  value = value >>> 8;
+  buf[offset++] = value;
+  value = value >>> 8;
+  buf[offset++] = value;
+  value = value >>> 8;
+  buf[offset++] = value;
+  return offset;
+}
+
+function writeU_Int24LE(
+    buf: Buffer,
+    value: number,
+    offset: number,
+    min: number,
+    max: number) {
+  value = +value;
+  validateOffset(offset, "offset", 0, buf.length);
+  checkInt(value, min, max, buf, offset, 2);
+
+  buf[offset++] = value;
+  value = value >>> 8;
+  buf[offset++] = value;
+  value = value >>> 8;
+  buf[offset++] = value;
+  return offset;
+}
+
+export default {
+  Buffer,
+  constants,
+  kMaxLength,
+  kStringMaxLength,
+  SlowBuffer,
+};
```

Comment:
Buffer is huge...

Reply:
Yes, yes it is. There are bigger ones tho.